features/completed-form-available-to-download.feature

```diff
@@ -0,0 +1,58 @@
+@e2e
+@javascript
+Feature:
+  As an ATOS admin
+  I want to be able to download completed forms
+  So I can triage an employees claim against their employer
+
+  Scenario: PDF format
+    Given I am making a claim
+    When I submit a completed Employment Tribunal form
+    Then the completed form in PDF format is available for ATOS to download and validate
+
+  Scenario: TXT format
+    Given I am making a claim
+    When I submit a completed Employment Tribunal form
+    Then the completed form in TXT format is available for ATOS to download and validate
+
+  Scenario: XML format
+    Given I am making a claim
+    When I submit a completed Employment Tribunal form
+    Then the completed form in XML format is available for ATOS to download and validate
+
+  Scenario: No representative
+    Given I am making a claim with no respresentative
+    When I submit a completed Employment Tribunal form
+    Then the completed form in PDF format is available for ATOS to download and validate
+
+  Scenario: One other person making claim
+    Given I am making a claim with one other person
+    When I submit a completed Employment Tribunal form
+    Then the completed form in PDF format is available for ATOS to download and validate
+
+  Scenario: Validate TXT file when uploading CSV data with 7 other people making the claim
+    Given I am making a claim with 7 other people
+    When I submit a completed Employment Tribunal form
+    Then the uploaded CSV data is formatted into a TXT format file and is available for ATOS to download
+
+  Scenario: Validate CSV file when uploading CSV data with 7 other people making the claim
+    Given I am making a claim with 7 other people
+    When I submit a completed Employment Tribunal form
+    Then the uploaded CSV data is formatted and available for ATOS to download
+
+  Scenario: Uploading RTF data when describing your claim in details
+    Given I am making a claim using a separate document
+    When I submit a completed Employment Tribunal form
+    Then the completed form in RTF format is available for ATOS to download and validate
+
+  Scenario: Making claim against 3 employers
+    Given I am making a claim against 3 employers
+    When I submit a completed Employment Tribunal form
+    Then the completed form in TXT format should contained 3 employers details
```

Comment:
In the other scenarios, we are using ' ... is available for ATOS to download' - i.e. we are saying that we are examining the file from the atos interface.  But, in this one it just talks about 'the completed form in txt format', not stating that you are looking at the file from the atos interface, but from 'somewhere unknown' !

Also, I think (but could be wrong !) the step such as 'Then the uploaded CSV data is formatted and available for ATOS to download' checks the contents of the file you are talking about based on the given input data - so is there any need to do a step that says 'should contain 3 employers details' ?

I must admit, it is more explicit though which is nice - just inconsistent with the others.

Reply:
Test is to check check that the txt file contained 3 employers details

Scenario: Making claim against 3 employers
    Given an employee making a claim against 3 employers
    When the completed form is submitted
    Then I can download the form and validate the TXT file contained 3 employers details